.tests/misc-diagrams

```diff
@@ -1,20 +1,4 @@
 #!/bin/bash
 
 diagram_dir="./docs/"
-diagrams=$(make --quiet -C "$diagram_dir" diagrams)
-
-status=0
-for diagram in $diagrams ; do
-	# Truncate the file extension (.png, .svg, etc)
-	source=${diagram%????}
-
-	source_date=$(git log -1 --date=unix --format="%cd" -- "$diagram_dir$source")
-	diagram_date=$(git log -1 --date=unix --format="%cd" -- "$diagram_dir$diagram")
-
-	if [[ $diagram_date -lt $source_date ]]; then
-		echo "This diagram is out-of-date: $diagram_dir$diagram. Please generate an"`
-		      `" updated diagram and commit it to include it with your pull request."
-		status=1
-	fi
-done
-exit $status
+cargo make --cwd "$diagram_dir" check-diagrams
```

Comment:
if i remember this script is invoked from another cargo-make flow.
so its cargo-make->script->cargo-make
might want to replace that to cargo-make->cargo-make which is really simply to do

```toml
[tasks.call-another-cargo-make]
command = "cargo"
args = ["make", "check-diagrams"]
cwd = "CHANGE TO RIGHT PATH"
```

Reply:
I'm having an issues

I've forced the script to fail. If I call the makefile from the docs directory it exits with an error as expected.
From the main makefile the build doesn't fail. 

The main task looks like:
```
command = "cargo"
args = ["make", "check-diagrams"]
cwd = "${ENARX_DOCS_DIR}"
```

```
[peaches@deepwell enarx]$ cargo make misc-diagrams
[cargo-make] INFO - cargo make 0.32.0
[cargo-make] INFO - Project: enarx
[cargo-make] INFO - Build File: Makefile.toml
[cargo-make] INFO - Task: misc-diagrams
[cargo-make] INFO - Profile: development
[cargo-make] INFO - Running Task: misc-diagrams
[cargo-make] INFO - Execute Command: "cargo" "make" "check-diagrams"
[cargo-make][1] INFO - Project: enarx
[cargo-make][1] INFO - Build File: Makefile.toml
[cargo-make][1] INFO - Task: check-diagrams
[cargo-make][1] INFO - Profile: development
[cargo-make][1] INFO - Running Task: workspace
[cargo-make][1] INFO - Build Done in 0 seconds.
[cargo-make] INFO - Build Done in 0 seconds.
```

Reply:
And oddly enough the automatic testing found errors where I expected them to be. Weird

Reply:
oh, its getting the workspace emulation env vars from parent process so it tries to run it as a workspace.
such a complex setup.
you can set in env block of your new makefile workspace emulation false.

Reply:
Ok! It works as expected now. Thank you!